### DISCUSSION TITLE
Shape smoothing

### DISCUSSION THREAD

**Discussion by Yelgurk** [lovyan03/LovyanGFX#184] [Q&A] [answered]
Does the library support shape smoothing? In large circles, square corners are conspicuous. I know that lvgl can, but its capabilities are very excessive for my tasks.

**Comment by lovyan03** [comment, marked as answer]
@Yelgurk 
That feature is not currently available. I would like to support it at some point, but I need to get the alpha blending feature in place first.
I'd like to implement it after the alpha blending feature is fully developed.
